<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventsWindow</class>
 <widget class="QDialog" name="EventsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="EventsFunctionsBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>590</width>
     <height>340</height>
    </rect>
   </property>
   <property name="title">
    <string>Events Functions</string>
   </property>
   <widget class="QGroupBox" name="CreateEventBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Create Event</string>
    </property>
    <widget class="QPushButton" name="CreateEventButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Event</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="DeleteEventBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Delete Event</string>
    </property>
    <widget class="QPushButton" name="DeleteEventButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Event</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="UpdateEventBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Update Event</string>
    </property>
    <widget class="QPushButton" name="UpdateEventButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Event</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ViewingEventsBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>420</width>
      <height>260</height>
     </rect>
    </property>
    <property name="title">
     <string>Viewing Events</string>
    </property>
    <widget class="QTableView" name="ViewingEventsTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>400</width>
       <height>230</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>550</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HorizontalLayout">
     <item>
      <spacer name="HorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="UpdateTheTableButton">
       <property name="text">
        <string>Update the table</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
